fenix/app/src/main/java/org/mozilla/fenix/home/topsites/TopSiteItemViewHolder.kt

```diff
@@ -132,11 +89,65 @@ class TopSiteItemViewHolder(
         }
     }
 
+    /**
+     * Binds a top site to the view. This will set the title, url, and image.
+     * setOnClickListener and setOnLongClickListener are also set here.
```

Comment:
nit: With `setOnClickListener` and `setOnLongClickListener` being methods they are not _set_ here.
This method does set a `OnClickListener`, `OnTouchListener` and `OnLongClickListener` but these details are not needed to be made public. Same as with setting the `title`, `url` and `image`.

Looking through the current state of the codebase it looks like we just avoided adding documentation for the `bind` method or went with something general like `Bind the data for displaying a particular top site and respond to user interactions`.